browse-datasets/README.md

```diff
@@ -1,32 +1,41 @@
-# Tutorial: Browse Datasets that have been backed up
+# Tutorial: Annotate content that’s already on the network
 
 _Note: This tutorial is a work in progress. Please add your feedback to [datatogether/learning](https://github.com/datatogether/learning/issues)!_
 
 ## Prerequisites
 
-* Some familiarity with the command line
-* [...]
+* Some interest in data
+* Optional (but valuable if you have it) domain expertise 
+* Enthusiasm!
 
 ## Learning Objectives
 
 After going through this tutorial you will know how to:
 
-* [...]
-* [...]
-* [...]
+* Add metadata to data you care about
 
 ****
 
-### Step 1.
+### Step 1. Add metadata
 
-* [...]
+* Return to your web browser, and signup/signin to http://archivers.co/
 
-### Step 2.
+* Navigate to the content page for `CAFO_Density.zip`: http://archivers.co/content/12208b35b5c819ef6a8bb5df4657acbed1fbf084f8efe3174b3896db0058d91e82e5
```

Comment:
Can we use this url instead? http://archivers.co/content/QmQU7RoBd8FWAkbePif7Xw5cjoL1Wb6wxCefdTuoA7XGa7

Reply:
OK, done in commit 4b0e9eb